src/visitors/transpileCssProp.js

```diff
@@ -0,0 +1,104 @@
+// Most of this code was taken from @satya164's babel-plugin-css-prop
+// @see https://github.com/satya164/babel-plugin-css-prop
+import { useCssProp } from '../utils/options'
+
+const getTag = node => {
+  if (typeof node.name === 'string') return node.name
+  if (node.type === 'JSXMemberExpression') {
```

Comment:
I usually prefer to use it like `t.isJSXMemberExpression(node)` instead of checking type manually

Reply:
Great call